sched/mqueue/mqueue.h

```diff
@@ -37,7 +37,7 @@
 
 #include <nuttx/mqueue.h>
 
-#if CONFIG_MQ_MAXMSGSIZE > 0
```

Comment:
Is it failing to compile when CONFIG_MQ_MAXMSGSIZE is not defined?

Reply:
it generate the compiler warning.